UnityProject/Assets/Scripts/Input System/InteractionV2/TileInteraction/ElectricalCableDeconstruction.cs

```diff
@@ -53,9 +53,10 @@ public override void ServerPerformInteraction(TileApply interaction)
 			// Electrocute the performer. If shock is painful enough, cancel the interaction.
 			ElectricityFunctions.WorkOutActualNumbers(ElectricalData.InData);
 			float voltage = ElectricalData.InData.Data.ActualVoltage;
-			var electrocutionSeverity = new Electrocution().ElectrocutePlayer(
-				interaction.Performer, interaction.WorldPositionTarget, "cable", voltage);
-			if (electrocutionSeverity > Electrocution.Severity.Mild) return;
+			var electrocution = new Electrocution(voltage, interaction.WorldPositionTarget, "cable");
+			var performerLHB = interaction.Performer.GetComponent<LivingHealthBehaviour>();
+			var severity = performerLHB.Electrocute(electrocution);
+			if (severity > ElectrocutionSeverity.Mild) return;
```

Comment:
I'm not clear what effect the severity has on the electrocution logic. If there needs to be different behavior based on that, why not put that into LHB / PlayerHealth?